### PR TITLE
Documentation changes for CSI Powerscale Driver v2.6.0

### PR DESCRIPTION
**Pull request by rajkumar-palani**
# Description
Documentation update for CSI CSI Powerscale which includes:

- Update of support matrix
- Update of CSI Powerscale driver version

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/583|

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No edits required.

**Review by nitesh3108 — APPROVED** [review]

File: content/docs/csidriver/installation/operator/_index.md [hunk 1]
```diff
@@ -405,7 +405,7 @@ spec:
 You can set the field ***replicas*** to a higher number than `1` for the latest driver versions.
 
 Note - The `image` field should point to the correct image tag for version of the driver you are installing.  
-For e.g. - If you wish to install v2.5.0 of the CSI PowerMax driver, use the image tag `dellemc/csi-powermax:v2.5.0`
+For e.g. - If you wish to install v2.6.0 of the CSI PowerMax driver, use the image tag `dellemc/csi-powermax:v2.6.0`
```
Comment (prablr79) [inline content/docs/csidriver/installation/operator/_index.md:408]: why there Powrmax driver mention here ? any reason ?

**Comment by prablr79** [comment]
@rajkumar-palani can you take care of branch conflicts and rebase the content ?

**Comment by rajkumar-palani** [comment]
> @rajkumar-palani can you take care of branch conflicts and rebase the content ?

@prablr79 - Sure Prasanna. we will do it once 1.5.1 changes merge with main. once done, will rebase this branch with main.

File: content/docs/csidriver/installation/operator/_index.md (hunk 1 above)
Comment (rajkumar-palani) [reply content/docs/csidriver/installation/operator/_index.md:408]: @prablr79 - This content is already present in the documentation and it is for an example. Please Refer. https://dell.github.io/csm-docs/docs/csidriver/installation/operator/
image

Comment (prablr79) [reply content/docs/csidriver/installation/operator/_index.md:408]: let us change Eg to Pscale. Thanks!

Comment (shanmydell) [reply content/docs/csidriver/installation/operator/_index.md:408]: @prablr79 : It is operator generic page, not Isilon page

**Review by shanmydell — APPROVED** [review]

**Review by prablr79 — APPROVED** [review]

**Review by Deepak-Ghivari — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**